src/_component/about-me/about-me.pug

```diff
@@ -0,0 +1,29 @@
+.column(span="row")
+  .section__title-wrapper
+    .section-title=data['about-me'].sectionTitle
+.column(span="row" height='10px')
+.column(span="row")
+  button.button__disc--red
+    i.icon__youtube 
+.column(span="row" height='10px')
+.column(span="row")
+  h1.about-me__name=data['about-me'].name
+.column(span="row")
+  h2.about-me__position=data['about-me'].position
+.column(span="row" height='10px')
+.column(span="row")
+  h3.about-me__info=data['about-me'].info
+.column(span="row")
+  .info-box__with-shadow
+    p.description=data['about-me'].descriptionList[0]
```

Comment:
loop the `data['about-me'].descriptionList` collection using `each`
ask for help if you don't know how to do it